local-tests/test_multiple_restarts.py

```diff
@@ -2,27 +2,28 @@
 import os
 import sys
 from os.path import abspath, join
-from time import sleep
+from time import sleep, ctime
 
 from chainrunner import Chain, Seq, generate_keys, check_finalized
 
+def printt(s): print(ctime() + ' | ' + s)
```

Comment:
what about `print = lambda s: print(ctime() + ' | ' + s)` :stuck_out_tongue:

Reply:
"To understand recursion, you must first understand recursion" ;)

Reply:
uhm, I thought that new `print` will be visible after declaration :|